kubernetes-csi-driver-nfs.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: kubernetes-csi-driver-nfs
+  version: 4.10.0
+  epoch: 0
+  description: This driver allows Kubernetes to access NFS server on Linux node
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-csi/csi-driver-nfs
+      tag: v${{package.version}}
+      expected-commit: 360cff75fa82720bbcc1eaba1194215d2937e750
+
+  - uses: go/build
+    with:
+      go-package: go
+      packages: ./cmd/nfsplugin
+      output: nfsplugin
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for matching with the upstream Dockerfile"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/
+          ln -sf /usr/bin/nfsplugin "${{targets.contextdir}}"/nfsplugin
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes-csi/csi-driver-nfs
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        nfsplugin --help
```

Comment:
non-blocking: if we can extend something more here that would be great. I think it will require a nfs up so not sure if we can do it here or not. maybe start nfs-server using test/daemon-check-output and do some tests around it. 💭

Reply:
That'd be great if we could do that but this is CSI Plugin and we need a real Kubernetes cluster to be able to test its functionality because we have to create a PersistentVolume like [this](https://github.com/kubernetes-csi/csi-driver-nfs/blob/b68ee7181160b12717c80c53667e2d33bf1edc7b/deploy/example/nfs-provisioner/nginx-pod.yaml#L17-L24) and then we should mount that PV to the Pod to see if we can write files to that directory and check NFS server that the file is there too.

Reply:
Then I saw this: https://github.com/kubernetes-csi/csi-driver-nfs/blob/b68ee71/test/sanity/run-test.sh then tried to setup an NFS server by following this guide: https://wiki.alpinelinux.org/wiki/Setting_up_an_NFS_server
but failed due to following error:

```bash
Exporting File System...
exporting *:/data
exportfs: /data does not support NFS export
Export validation failed, exiting...
```

so I added a simple test with grep.